baldur/src/main/java/org/opencadc/baldur/PermissionsConfig.java

```diff
@@ -198,11 +201,34 @@ private void init() {
                 tmp.add(next);
                 log.debug("Added permission entry: " + next);
             }
+
+            // get the Grant timeout
+            List<String> timeout = allProps.getProperty(KEY_EXPIRY_TIME);
+            if (timeout.size() == 0) {
+                throw new IllegalStateException("no values for key " + KEY_EXPIRY_TIME + " in " + PERMISSIONS_PROPERTIES);
+            }
+            if (timeout.size() > 1) {
+                throw new IllegalStateException("multiple values for key " + KEY_EXPIRY_TIME + " in "
+                    + PERMISSIONS_PROPERTIES + " where single value expected");
+            }
+            try {
```

Comment:
When used on thwe left-hand side of an assignment, I like to see
```
this.expiryDate =
```
so that when looking at code without an IDE I can tell that's a member variable assignment.

Reply:
The member variables in PermissionsConfig are static.

Reply:
ugh. I had not noticed that earlier. That severely limits how people can deploy instances of this and while we are currently targetting a container deployment so only one application per JVM the code could be deployed in other ways. Please change these to non-static.

Reply:
Updated the member variables to be non-static. I did have to remove the caching that made use of the static variables. Do we need a new caching method?